python/taichi/math/mathimpl.py

```diff
@@ -0,0 +1,399 @@
+"""
+Math functions for glsl-like functions and other stuff.
+"""
+from math import e, pi
+
+import taichi as ti
+
+
+@ti.func
+def mix(x, y, a):
```

Comment:
```suggestion
def lerp(x, y, a):
```

Reply:
I believe that `lerp` is a more common name. :)

Reply:
I guess `lerp` is used mostly for HLSL users? It's not in the GLSL spec: 
https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/